CMakeLists.txt

```diff
@@ -409,6 +409,9 @@ endif(WIN32 AND NOT MSVC)
 
 #Unix paths
 if(CMAKE_HOST_UNIX AND NOT CMAKE_HOST_APPLE)
+  set(CPACK_PACKAGE_CONTACT "contact@codisec.com")
+  set(CPACK_DEBIAN_PACKAGE_SECTION "utils")
+  set(CPACK_DEBIAN_PACKAGE_DEPENDS "libqt5core5a (>= 5.5.1), libqt5gui5 (>= 5.5.1), libqt5widgets5 (>= 5.5.1), libqt5network5 (>= 5.5.1), zlib1g (>= 1:1.2.8)")
```

Comment:
right now it is not required, next step is server script installation and inserting this dependency